src/components/ThumbnailImage/ThumbnailImage.vue

```diff
@@ -15,27 +15,47 @@
     <LazyLoadImage
       :src="src"
       :alt="alt"
-      class="group-hover:scale-110 w-full h-full object-cover transition-all"
+      class="group-hover:scale-110 w-full h-full object-cover transition-all group-hover:opacity-100 opacity-80"
     />
+    <div
+      v-if="isVideo || isAudio"
+      class="backdrop-blur-md bg-transparent-white-500 text-neutral-900 flex absolute bottom-1 right-1 z-10 rounded-full justify-center items-center p-1 w-6 h-6"
+    >
+      <AudioIcon v-if="isAudio" />
+      <VideoIcon v-if="isVideo" />
+    </div>
     <slot />
   </component>
 </template>
 <script setup lang="ts">
+import { computed } from "vue";
 import LazyLoadImage from "@/components/LazyLoadImage/LazyLoadImage.vue";
-import ArrowForwardIcon from "@/icons/ArrowForwardIcon.vue";
+import { ArrowForwardIcon, AudioIcon, VideoIcon } from "@/icons";
 
-withDefaults(
+const props = withDefaults(
   defineProps<{
     src: string;
     alt: string | null;
     href?: string;
     isActive?: boolean;
+    fileType?: string | undefined;
   }>(),
   {
     href: undefined,
     isActive: false,
+    fileType: undefined,
   }
 );
+
+const isVideo = computed((): boolean => {
```

Comment:
If we’re going to use file extensions here, they should mirror the allow list types from audiohandler/moviehandler.